owncloudApp/src/main/java/com/owncloud/android/authentication/AccountAuthenticator.java

```diff
@@ -97,8 +97,7 @@ public Bundle addAccount(AccountAuthenticatorResponse response,
             try {
                 validateAccountType(accountType);
             } catch (AuthenticatorException e) {
-                Timber.e(e, "Failed to validate account type " + accountType + ": " + e.getMessage());
-                e.printStackTrace();
+                Timber.e(e, "Failed to validate account type %s", accountType);
```

Comment:
We are duplicating code here, please create a method with:

```
try {
    validateAccountType(accountType);
} catch (AuthenticatorException e) {
    Timber.e(e, "Failed to validate account type %s", accountType);
    return e.getFailureBundle();
}
```

Reply:
I think that it is out of this pr scope which is to stop using printStacktrace.

BTW, we are going to refactor this class in the short term future with the new arch. So we can face this kind of problems there. Can't we?

Reply:
I try to use the boy scout rule (leave the code better than you found it) if the changes are tiny. I thought this was a good example but it looks good to me.